### ISSUE TITLE
Am I the only one having lag?

### ISSUE THREAD

**Issue by farooqkz** [MT-CTF/capturetheflag#1106]
I think it would be nice to add a command to check if I am the only one having lag or it's everyone.

**Comment by Ottobunny** [comment]
I am also having lots of lag, and a TON of server lag too. My PC is pretty powerfull, it has an i7 and readon pro graphics, and I get way less FPS than on playing on the JMA CTF server, witch is hosted from germany, when I live in the US lol.

**Comment by farooqkz** [comment]
For this what I have in mind is a command which iterates over all online players and check their RTT and shows average and worst metrics back to the user calling this command. What do you think @LoneWolfHT ?

**Comment by farooqkz** [comment]
> I have never experienced freezes on the sever, which is related to your computer powered and not the sever. The only times I get low FPS is when I'm on my laptop with shaders and shadows enabled. I have experienced internet lag but the reason behind the lag is always my own internet. Once I connect to Germany or UK using a VPN, the lag disappears.

The reason of network lag is *not* always client side or connection. Sometimes, not very frequent tho, the problem with lag is the server or its network.

**Comment by a-blob** [comment]
> 
> The reason of network lag is _not_ always client side or connection. Sometimes, not very frequent tho, the problem with lag is the server or its network.

Yes, this is possible and probably exists, but the effect is so minimal I rarely experience it. Most of the time I just refresh my own network and the lag would be gone.

**Comment by LoneWolfHT** [comment]
> For this what I have in mind is a command which iterates over all online players and check their RTT and shows average and worst metrics back to the user calling this command. What do you think @LoneWolfHT ?
> 
> Edit: This plan is given that most players will have high RTTs when the server is lagging.

That sounds fine to me
